### PR TITLE
Jumper T-20 2400 and 900 target defines

### PR DESCRIPTION
**Pull request by SunjunKim**
New targets for Jumper T-20 internal 2400 and 900 modules.

Power values have been calibrated using the supplied samples.

### CODE REVIEW

**Review by pkendall64 — APPROVED** [review]

File: src/hardware/targets.json [hunk 1]
```diff
@@ -1234,6 +1242,16 @@
...
             }
         },
+        "tx_900": {
+            "t-20": {
+                "product_name": "Jumper AION T-20 900M TX",
+                "lua_name": "T-20 900 TX",
+                "layout_file": "Jumper T-20 900.json",
+                "upload_methods": ["uart", "wifi", "etx"],
+                "platform": "esp32",
+                "firmware": "Unified_ESP32_2400_TX"
```
Comment (JyeSmith) [inline src/hardware/targets.json:1252]: Did you flash this?

File: src/hardware/TX/Jumper T-20 2400.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "serial_rx": 3,
+    "serial_tx": 1,
+    "radio_busy": 21,
+    "radio_dio1": 4,
+    "radio_dio2": 22,
```
Comment (JyeSmith) [inline src/hardware/TX/Jumper T-20 2400.json:6]: This does not appear to be connected on the schematic?

File: src/hardware/targets.json (hunk 1 above)
Comment (SunjunKim) [reply src/hardware/targets.json:1252]: That's a mistake! 900 json itself is tested, but not targets.json.. I'll correct soon

File: src/hardware/TX/Jumper T-20 2400.json (hunk 2 above)
Comment (SunjunKim) [reply src/hardware/TX/Jumper T-20 2400.json:6]: checked.

File: src/hardware/targets.json (hunk 1 above)
Comment (SunjunKim) [reply src/hardware/targets.json]: Typo corrected. Tested with actual flashing process (wifi & etx passthrough)

**Review by JyeSmith — APPROVED** [review]

**Pull request merged**